Codigo/InvUsuario.bas

```diff
@@ -687,7 +687,7 @@ On Error GoTo GetSlotForItemInInvetory_Err
 100 Dim i As Integer
     
 102 For i = 1 To UserList(UserIndex).CurrentInventorySlots
-104    If UserList(UserIndex).invent.Object(i).objIndex = 0 Then
+104    If UserList(UserIndex).Invent.Object(i).objIndex = 0 And GetSlotForItemInInvetory = -1 Then
```

Comment:
no, we are looping here so every time we find a new slot we set it to a new position if we have multiple empty slot we will add the item to the last one instead of the first empty slot
we cant just exit when we find the first empty slot because we need to see if we have a slot with the same item to stack the item